### PR TITLE
Mostly routes

### PR DESCRIPTION
**Pull request by neworderofjamie**
- Most important thing is a small fix to ``RouteContinuous`` so you can make repeated calls to ``load``. 
- Added a method to ``RouteContinuous`` to generate Levy walks (based on the wikipedia entry so, if you have any deeper knowledge, it'd be appreciated)
- Fixed a minor windows issue where the wrong library directory was used when making debug builds

Also, in #183, you introduced a fix for the duplicate path-less libraries being added to the Visual Studio project. Adding the library directory fixes this for release but these still need manually removing in debug as, whatever is adding them, doesn't take into account the debugness and adds the versions with the wrong names (which don't exist in the debug library directory). Not sure if that's a helpful clue as to the underlying issue or not...

### CODE REVIEW

File: src/antworld/route_ardin.cc [hunk 1]
```diff
@@ -275,7 +275,7 @@ void RouteArdin::render(const Pose2<meter_t, degree_t> &pose, meter_t height) co
 
     glBindVertexArray(m_OverlayVAO);
 
-    glTranslatef(static_cast<GLfloat>(pose.x().value()), static_cast<GLfloat>(pose.y().value()), 0.0f);
+    glTranslatef(static_cast<GLfloat>(pose.x().value()), static_cast<GLfloat>(pose.y().value()), 0.01f);
```
Comment (alexdewar) [inline src/antworld/route_ardin.cc:278]: I'm guessing you are hardcoding the height to 10cm here, right? Shouldn't z be set to ``height`` instead?

File: src/antworld/route_continuous.cc [hunk 2]
```diff
@@ -235,18 +161,17 @@ void RouteContinuous::render(const Pose2<meter_t, degree_t> &pose, meter_t heigh
...
     if(m_RouteNumPoints > 0) {
         glBindVertexArray(m_RouteVAO);
-
         glDrawArrays(GL_LINE_STRIP, 0, m_RouteNumPoints);
     }
 
     glBindVertexArray(m_OverlayVAO);
 
-    glTranslatef(static_cast<GLfloat>(pose.x().value()), static_cast<GLfloat>(pose.y().value()), 0.0f);
+    glTranslatef(static_cast<GLfloat>(pose.x().value()), static_cast<GLfloat>(pose.y().value()), 0.01f);
```
Comment (alexdewar) [inline src/antworld/route_continuous.cc:174]: See above

File: src/antworld/route_ardin.cc (hunk 1 above)
Comment (neworderofjamie) [reply src/antworld/route_ardin.cc:278]: So, (deprecated but convenient) OpenGL provides you with matrix stack onto which you can pop and push transforms. At the top of these functions we push the height z-transform before doing any (essentially 2D) drawing. This additional translate is just to make sure the arrow renders above the route

Comment (alexdewar) [reply src/antworld/route_ardin.cc:278]: Ahhh I see. Makes sense. There's so much I don't know about OpenGL :laughing:

**Pull request merged**